OpenRA.Mods.Common/Traits/FireWarheadOnDeath.cs

```diff
@@ -21,7 +21,7 @@ public enum ExplosionType { Footprint, CenterPosition }
 	public enum DamageSource { Self, Killer }
 
 	[Desc("This actor explodes when killed.")]
-	public class ExplodesInfo : ConditionalTraitInfo, Requires<IHealthInfo>
+	public class FireWarheadOnDeathInfo : ConditionalTraitInfo, Requires<IHealthInfo>
```

Comment:
```suggestion
	public class FireWarheadsOnDeathInfo : ConditionalTraitInfo, Requires<IHealthInfo>
```

since we fire multiple warheads, that also solves the nit about `Weapon`.

Reply:
we fire multiple only on footprint type, but I suppose that counts